### PR TITLE
Search feature improvements of UI logic

### PR DESCRIPTION
**Pull request by marcin-sieminski**
Minor bugfixes concerning search features logic - preventing incorrect inputs + extracted GetTextToSearch method.

### CODE REVIEW

File: DrinkManagerConsole/SearchDrinkConsoleUi.cs [hunk 1]
```diff
@@ -21,45 +21,81 @@ public static void StartSearch(List<Drink> drinksList, SearchCriterion searchCri
...
+                    }
+                } while (incorrectInputEndSearch);
+                
             } while (continueSearch);
         }
 
+        private static string GetTextToSearch(SearchCriterion searchCriterion)
+        {
+            string textToSearch;
+            var incorrectInput = true;
```
Comment (wojciechkapko) [inline DrinkManagerConsole/SearchDrinkConsoleUi.cs]: I'm not sure if this boolean value is needed, maybe we could do string.IsNullOrWhiteSpace(textToSearch) in the while condition ?

File: DrinkManagerConsole/SearchDrinkConsoleUi.cs [hunk 2]
```diff
@@ -21,45 +21,81 @@ public static void StartSearch(List<Drink> drinksList, SearchCriterion searchCri
...
+            do
+            {
+                Console.Write($"\nEnter a drink {searchCriterion.ToString().ToLower()} to find: ");
+                textToSearch = Console.ReadLine();
+                if (textToSearch != "")
```
Comment (wojciechkapko) [inline DrinkManagerConsole/SearchDrinkConsoleUi.cs]: This if statement would not be needed too

File: DrinkManagerConsole/SearchDrinkConsoleUi.cs (hunk 1 above)
Comment (marcin-sieminski) [reply DrinkManagerConsole/SearchDrinkConsoleUi.cs]: Nice solution - implemented & pushed.

**Comment by marcin-sieminski** [comment]
Proposed improvement implemented and pushed. PR ready for re-review.

**Review by wojciechkapko — APPROVED** [review]

**Pull request merged**